typescript/infra/scripts/check/check-utils.ts

```diff
@@ -207,9 +207,12 @@ export async function getGovernor(
       warpRouteId,
       registryUris,
     );
-    const warpAddresses = registryUris
-      ? await getWarpAddressesFromMergedRegistry(warpRouteId, registryUris)
-      : getWarpAddresses(warpRouteId);
+    const warpAddresses = await getWarpAddressesFrom(warpRouteId, registryUris);
+    const warpDeployConfig = await findWarpDeployConfig(
```

Comment:
what's the difference between this and `config` in line 204?

Reply:
That continues the thread on `HypTokenRouterConfig` vs `WarpRouteDeployConfig`. Having `config` from line 204 is probably enough. Let me check. Maybe this whole PR can be made much smaller.

Reply:
Removed this var and the new `findWarpDeployConfig`